data.sql

```diff
@@ -38,4 +38,39 @@ UPDATE animals SET owner_id = 1 WHERE id = 1;
 UPDATE animals SET owner_id = 2 WHERE id = 2 OR id = 3;
 UPDATE animals SET owner_id = 3 WHERE id = 4 OR id = 6;
 UPDATE animals SET owner_id = 4 WHERE id = 5 OR id = 7 OR id = 10;
-UPDATE animals SET owner_id = 5 WHERE id = 8 OR id = 9;
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE id = 8 OR id = 9;
+
+-- vets data
+INSERT INTO vets (name, age, date_of_graduation) 
+        VALUES 
+            ('William Tatcher', 45, '2000-04-23'), 
+            ('Maisy Smith', 26, '2019-01-17'), 
+            ('Stephanie Mendez', 64, '1981-05-04'), 
+            ('Jack Harkness', 38, '2008-06-08');
+
+-- specialization data
+INSERT INTO specialization (vet_id, species_id) VALUES (1, 1), (3, 2), (3, 1), (4, 2);
+
+-- visits data
+INSERT INTO visits (animals_id, vets_id, date_of_visit) 
+            VALUES 
+                (1, 1, '2020-05-24'), 
+                (1, 3, '2020-07-22'), 
+                (2, 4, '2021-02-02'), 
+                (3, 2, '2020-01-05'),
+                (3, 2, '2020-03-08'),
+                (3, 2, '2020-05-14'),
+                (4, 3, '2021-05-04'),
+                (5, 4, '2021-02-24'),
+                (6, 2, '2019-12-21'),
+                (6, 1, '2020-06-10'),
```

Comment:
- [ ] I believe you made a typo here on line 66. The correct month should be August (08). You did use (06).